superduperdb/backends/ibis/query.py

```diff
@@ -620,12 +620,19 @@ class Table(Component):
     schema: Schema
     primary_id: str = 'id'
 
-    def __post_init__(self):
-        super().__post_init__()
+    def __post_init__(self, artifacts):
+        super().__post_init__(artifacts)
```

Comment:
Good idea.

Reply:
Unfortunately this won't work, since the first time a users passes, the `_fold` is usually not specified. But on saving/ deserialization it is there. So the `assert` breaks the saving mechanism.